test1/CellViewController.swift

```diff
@@ -12,139 +12,184 @@ import SDWebImage
 class NikolaAwesomeLabel: UILabel{
     override init(frame: CGRect){
         super.init(frame: frame)
-        self.backgroundColor = .white
         self.font = .systemFont(ofSize: 20)
-        self.textAlignment = .left
         self.numberOfLines = 0
         self.lineBreakMode = .byWordWrapping
     }
-    
+
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 }
 
-class CellViewController: UIViewController{
+class CellViewController: UIViewController, UITableViewDelegate, UITableViewDataSource{
     
-
     var person: CharacterModel
-    //let tableView = UITableView()
     
-    init(person: CharacterModel){
-        self.person = person
-        super.init(nibName: nil, bundle: nil)
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return person.episode.count
     }
     
-    required init?(coder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
     }
     
-    
-    override func viewDidLoad() {
-        super.viewDidLoad()
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let label = UILabel()
+        label.font = UIFont.boldSystemFont(ofSize: 15)
+        label.text = "Episodes:"
+        label.textAlignment = .center
+
+        let view = UIView()
         view.backgroundColor = .white
-        //view.addSubview(tableView)
-//      tableView.delegate = self
-//      tableView.dataSource = self
-        makeStackView()
+        view.addSubview(label)
+
+        label.snp.makeConstraints{make in
+            make.leading.trailing.bottom.equalToSuperview().inset(20)
+            make.top.equalToSuperview()
+        }
+
+        return view
     }
     
-    lazy var image: UIImageView = {
-        let temp = UIImageView()
-        temp.sd_setImage(with: URL(string: person.image))
-        return temp
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "character",for: indexPath)
+        cell.selectionStyle = .none
+        cell.textLabel?.text = person.episode[indexPath.row]
+        cell.textLabel?.textAlignment = .center
+        return cell
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 30
```

Comment:
this is not needed if you create the layout of the cells correctly. they can draw themselves.